src/Azure.Deployments.Extensibility.Extensions.Kubernetes/EmbededResources/ApiCatalog/K8sApiCatalog.csv

```diff
@@ -0,0 +1,94 @@
+﻿Group,Version,Kind,Plural,Namespaced,MajorMinorServerVersions

```

Comment:
The file was generated by a console app which will be included in a follow-up PR.